guide/14-deep-learning/text-classification-workflow.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<p>"Transformers in NLP are novel architectures that aims to solve sequence-to-sequence tasks while handling long-range dependencies with ease." -&gt; I think this line can either be elaborated on what 'sequence-to-sequence' tasks are and what long range deps are. If that is too complex within the confines of a single notebook, please hyperlink to a reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Added appropriate hyperlinks to <strong>sequence-to-sequence</strong> and <strong>long-range dependencies</strong> terms</p>